src/components/settings/Settings.cpp

```diff
@@ -8,7 +8,6 @@ Settings::Settings(Pinetime::Controllers::FS& fs) : fs {fs} {
 }
 
 void Settings::Init() {
-
```

Comment:
Accidental whitespace removal?
Or if this is on purpose, also remove from below

Reply:
Removed the other whitespace